libs/designsystem/src/lib/helpers/deep-copy.ts

```diff
@@ -1,3 +1,13 @@
-export function deepCopy(obj: Object) {
-  return JSON.parse(JSON.stringify(obj));
+import _deepCopy from 'ts-deepcopy';
+
+/**
+ * Do a deep copy of object to remove references
+ *
+ * Do so while retaining functions.
+ *
+ * @param obj
+ * @returns
```

Comment:
Do you have anywhere, where I can find more on this convention in comments? 
Can't seem to find anything, but I do like it 😄

Reply:
I added a link to a stackoverflow answer describing the problem, as I mentioned earlier, I used this package as a replacement of [lodash.cloneDeep](https://lodash.com/docs/4.17.15#cloneDeep)